utils/apiUtil.js

```diff
@@ -139,6 +140,22 @@ module.exports = {
     };
   },
 
+  getJWTData(path_to_credentials_json) {
+    const credentials = fs.readFileSync(path_to_credentials_json, 'utf8');
+    const api_key = JSON.parse(credentials)['api_key'];
+
+    data = api_key.split('.'); // [0] headers, [1] payload, [2] signature
+    payload = data[1];
+    padding = "=".repeat(4 - payload.length % 4);
+    decoded_data = Buffer.from((payload + padding), 'base64').toString();
+    // If the decoded data doesn't contain a nonce, that means the refresh token has expired
+    const target_environment = JSON.parse(decoded_data)['iss'].split('/')[2];
+    return {
+        'api_key': api_key,
+        'target_environment': target_environment
+    };
+  },   
+
```

Comment:
Looks like there are two `token.txt` files in this branch:
> suites/load-testing/token.txt 
and
> utils/token.txt 

We gotta remove them.